src/generic/fontdlgg.cpp

```diff
@@ -376,6 +376,7 @@ void wxGenericFontDialog::CreateWidgets()
     if (ShowToolTips())
         itemChoice7->SetToolTip(_("The font family."));
     itemBoxSizer5->Add(itemChoice7, 0, wxALIGN_LEFT|wxALL, 5);
+    m_familyChoice = itemChoice7;
```

Comment:
I think it would be nice to get rid of `itemChoiceN` entirely. I.e. if the goal is to make this code more clear, it would be better to avoid such auto-generated variable names and just use `m_familyChoice` everywhere.

Of course, it could be done in a subsequent commit.

Reply:
I agree it would be nice, however if we replace relatively few itemChoice* variables with class members it will look even more confusing. I would leave it like this for now.

Reply:
Just to be clear, I meant to replace just the variables that we already have members for. I.e. here I don't see any point in `itemChoice7` existence, we should just create `m_familyChoice` directly.

It's not terribly important, but if you're cleaning up this code anyhow, why not do this too?